Let me take a look at these failures for Ruby 2.7 and 3.0.

https://buildkite.com/rails/rails/builds/77864#a957003c-605f-45e7-a3d8-e02a5a78e23e

```
Traceback (most recent call last):
	9: from bug_report_templates/action_mailbox_gem.rb:5:in `<main>'
	8: from /usr/local/bundle/gems/bundler-2.2.9/lib/bundler/inline.rb:54:in `gemfile'
	7: from /usr/local/bundle/gems/bundler-2.2.9/lib/bundler/settings.rb:115:in `temporary'
	6: from /usr/local/bundle/gems/bundler-2.2.9/lib/bundler/inline.rb:70:in `block in gemfile'
	5: from /usr/local/bundle/gems/bundler-2.2.9/lib/bundler/runtime.rb:26:in `setup'
	4: from /usr/local/bundle/gems/bundler-2.2.9/lib/bundler/runtime.rb:26:in `map'
	3: from /usr/local/bundle/gems/bundler-2.2.9/lib/bundler/spec_set.rb:150:in `each'
	2: from /usr/local/bundle/gems/bundler-2.2.9/lib/bundler/spec_set.rb:150:in `each'
	1: from /usr/local/bundle/gems/bundler-2.2.9/lib/bundler/runtime.rb:31:in `block in setup'
/usr/local/bundle/gems/bundler-2.2.9/lib/bundler/runtime.rb:302:in `check_for_activated_spec!': You have already activated timeout 0.1.0, but your Gemfile requires timeout 0.1.1. Since timeout is a default gem, you can either remove your dependency on it or try updating to a newer version of bundler that supports timeout as a default gem. (Gem::LoadError)
```

--- a/guides/bug_report_templates/action_mailbox_gem.rb
+++ b/guides/bug_report_templates/action_mailbox_gem.rb
@@ -10,6 +10,11 @@ gemfile(true) do
   # Activate the gem you are reporting the issue against.
   gem "rails", "6.1.0"
   gem "sqlite3"
+  if RUBY_VERSION >= "3.1"
+    # net-smtp, net-imap and net-pop were removed from default gems in Ruby 3.1, but is used by the `mail` gem.
+    # So we need to add them as dependencies until `mail` is fixed: https://github.com/mikel/mail/pull/1439
+    gem "net-smtp", require: false
+  end
 end
 
 require "active_record/railtie"

--- a/guides/bug_report_templates/action_mailbox_main.rb
+++ b/guides/bug_report_templates/action_mailbox_main.rb
@@ -9,6 +9,11 @@ gemfile(true) do
 
   gem "rails", github: "rails/rails", branch: "main"
   gem "sqlite3"
+  if RUBY_VERSION >= "3.1"
+    # net-smtp, net-imap and net-pop were removed from default gems in Ruby 3.1, but is used by the `mail` gem.
+    # So we need to add them as dependencies until `mail` is fixed: https://github.com/mikel/mail/pull/1439
+    gem "net-smtp", require: false
+  end
 end
 
 require "active_record/railtie"

